configs/tof/tof_spynet-sintel-wobn-clean_1xb1_vimeo90k-triplet.py

```diff
@@ -1,7 +1,8 @@
-_base_ = '../_base_/models/base_tof_vfi_nobn_1xb1_vimeo90k_triplet.py'
+_base_ = '../_base_/models/base_tof.py'
 
-experiment_name = 'tof_vfi_spynet_clean_nobn_1xb1_vimeo90k'
```

Comment:
What does `vfi` mean?

Reply:
task name: video frame interpolation. 
For now, we remove task name in config filename